test/Common/DurableClientBaseTests.cs

```diff
@@ -31,9 +28,28 @@ public async Task StartNewAsync_is_calling_overload_method()
             var instanceId = Guid.NewGuid().ToString();
             const string functionName = "sampleFunction";
             var durableOrchestrationClientBaseMock = new Mock<IDurableOrchestrationClient> { CallBase = true };
-            durableOrchestrationClientBaseMock.Setup(x => x.StartNewAsync(functionName, string.Empty, null)).ReturnsAsync(instanceId);
+            durableOrchestrationClientBaseMock.Setup(x => x.StartNewAsync<object>(functionName, string.Empty, null)).ReturnsAsync(instanceId);
 
-            var result = await durableOrchestrationClientBaseMock.Object.StartNewAsync(functionName, null);
+            var result = await durableOrchestrationClientBaseMock.Object.StartNewAsync(functionName);
+            result.Should().Be(instanceId);
+
+            result = await durableOrchestrationClientBaseMock.Object.StartNewAsync<object>(functionName, null);
+            result.Should().Be(instanceId);
+        }
+
+        [Fact]
+        [Trait("Category", PlatformSpecificHelpers.TestCategory)]
+        public async Task StartNewAsync_is_calling_overload_method_with_specific_instance_id()
```

Comment:
NIT: if this test, and the other test in this file, could be renamed to match the naming scheme of our other tests, that would be great.

Reply:
can you elaborate? I kept the name for one, and added a new one that _seemed_ to follow the same convention....

![image](https://user-images.githubusercontent.com/20270743/67590966-1a582600-f711-11e9-93eb-ef47790cbdfb.png)

happy to make changes, though!

Reply:
I now realize we don't have a firm convention. I have a preference towards Capitalizing the first letter after each `_`, but that is not consistent across our tests, so I won't make you address that in this PR. 

Making it more consistent is for another PR :).